core/modules/installer/installer.module

```diff
@@ -13,6 +13,7 @@ function installer_permission() {
       'title' => t('Install and update modules, themes & layouts'),
       'description' => t('Allows the user to browse for and install new modules, themes and layouts.'),
       'restrict access' => TRUE,
+      'warning' => t('Allows people to add <em>any</em> functionality to Backdrop.'),
```

Comment:
```suggestion
      'warning' => t('Allows people to add <em>any</em> functionality to the site.'),
```

Reply:
Not sure about this... Installing or updating modules isn't really about *changing existing functionality*. I feel like that's more when you give access to admin forms, etc.